.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -2,7 +2,7 @@
 name: Enhancement
 about: New feature or enhancement
 title: New Feature
-labels: enhancement
+labels: enhancement, triage
```

Comment:
Would this need to be enclosed in square brackets, e.g.: `[enhancement, triage]` ? The other PR does it that way.